keps/sig-node/0035-20190130-topology-manager.md

```diff
@@ -185,7 +185,10 @@ Kubelet components on a container by container basis.
 If the hints are not compatible, the Topology Manager may choose to
 reject the pod. Behavior in this case depends on a new Kubelet configuration
 value to choose the topology policy. The Topology Manager supports four
-policies: `none`(default), `best-effort`, `restricted` and `single-numa-node`. 
+policies: `none`(default), `best-effort`, `restricted`, `single-numa-node` and `pod-level-single-numa-node`.
+
+NOTE: All policies work on per container basis, the only exception is `pod-level-single-numa-node`.
+This policy attempts to find a NUMA node, from which all containers in a pod can have resources allocated. Then if Topology Manager found proper NUMA node, the pod will be admitted. Otherwise the pod will be rejected.
```

Comment:
The obvious concern to me is that it seems like trying to align all resources across all containers for a pod on a single NUMA node would exponentially increase the likelihood of the pod failing. 

As this is something that we've seen previously with `restricted` and `single-numa-node` policies, I'd be interested to hear what other people think (ie Is the enhancement of the extra policy worth the compromise of increased pod failure risk?).

Reply:
I left major usecase of new police on the below comment.
https://github.com/kubernetes/enhancements/pull/1752#issuecomment-629168507